pkgs/development/libraries/hyperscan/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib, stdenv, fetchFromGitHub, cmake, ragel, python27
+, boost
+}:
+
+assert stdenv.isx86_64;  
```

Comment:
should this not be moved towards a platform restriction in the `meta` attribute?